editor/code_editor.cpp

```diff
@@ -771,6 +771,14 @@ void CodeTextEditor::set_error(const String &p_error) {
 void CodeTextEditor::_update_font() {
 
 	text_editor->add_font_override("font", get_font("source", "EditorFonts"));
+
+	Ref<Font> status_bar_font = get_font("status_source", "EditorFonts");
+	int count = status_bar->get_child_count();
+	for (int i = 0; i < count; i++) {
+		Control *n = Object::cast_to<Control>(status_bar->get_child(i));
+		if (n)
+			n->add_font_override("font", status_bar_font);
```

Comment:
I could be wrong but doesn't Godot also inherit font overrides.

Or is that not the case for overrides.

If so than it's all good!

Reply:
@toger5 good point.
@Chaosus does not it work to simply `status_bar->add_font_override("font", status_bar_font);`, instead of looping with children nodes?

Reply:
Good point, will test now

Reply:
@volzhs Its doesnt work

Reply:
@toger5 Status bar uses own font and this font need to be updated like a text_editor in previous line